### PR TITLE
new example 4/workaround

### PR DESCRIPTION
**Pull request by trudolf-msft**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by trudolf-msft** [comment]
See https://github.com/Azure/azure-powershell/issues/8371#:~:text=Description%20Set-AzNetworkSecurityRuleConfig%20does%20not%20produce%20an%20output%20that,sure%20what%20the%20purpose%20of%20the%20cmdlet%20is.

**Comment by isra-fel** [comment]
Thanks for the contribution @trudolf-msft !
BTW, please visit https://aka.ms/AzureGitHub and join Azure orgnization.

File: src/Network/Network/help/Set-AzNetworkSecurityRuleConfig.md [hunk 1]
```diff
@@ -67,6 +67,15 @@ Updates a network security rule configuration for a network security group. (aut
...
 ```
 
+### Example 4
+
+Updates a network security rule configuration for a network security group (Source IP address)
+
+$nsg = Get-AzNetworkSecurityGroup -ResourceGroupName "MyResource" -Name "MyNsg"
+($nsg.SecurityRules | Where-Object {$_.Name -eq "RuleName"}).SourceAddressPrefix = ([System.String[]] @("xxx.xxx.xxx.xxx"))
+$nsg | Set-AzNetworkSecurityGroup | Get-AzNetworkSecurityRuleConfig -Name "RuleName"
+```
```
Comment (isra-fel) [inline src/Network/Network/help/Set-AzNetworkSecurityRuleConfig.md:77]: Please use "```powershell" and "```" to wrap your example

File: src/Network/Network/help/Set-AzNetworkSecurityRuleConfig.md [hunk 2]
```diff
@@ -67,6 +67,15 @@ Updates a network security rule configuration for a network security group. (aut
 Set-AzNetworkSecurityRuleConfig -Access Allow -Description 'Allow RDP' -DestinationAddressPrefix * -DestinationPortRange 3389 -Direction Inbound -Name 'rdp-rule' -NetworkSecurityGroup <PSNetworkSecurityGroup> -Priority 1 -Protocol Tcp -SourceAddressPrefix 'Internet' -SourcePortRange *
 ```
 
+### Example 4
+
+Updates a network security rule configuration for a network security group (Source IP address)
+
+$nsg = Get-AzNetworkSecurityGroup -ResourceGroupName "MyResource" -Name "MyNsg"
```
Comment (isra-fel) [inline src/Network/Network/help/Set-AzNetworkSecurityRuleConfig.md:74]:
```suggestion
```powershell
$nsg = Get-AzNetworkSecurityGroup -ResourceGroupName "MyResource" -Name "MyNsg"
```
Missed the starting mark of the code block.

**Comment by msJinLei** [comment]
kindly ping @trudolf-msft

**Comment by msJinLei** [comment]
Depend on https://github.com/Azure/azure-powershell/pull/16583 to merge

**Review by msJinLei — APPROVED** [review]

**Pull request merged**